src/database/people/index.ts

```diff
@@ -0,0 +1,115 @@
+import dbConnection, { tableNames, execute } from '../connection';
+
+const R = require('ramda');
```

Comment:
I don't think we are supposed to be importing like this. 
Should be like this: `import * as R from 'ramda';`

Reply:
no problem, but what's the difference?

Reply:
I'm not sure but Brad says so :P